maven-resolver-provider/src/main/java/org/apache/maven/repository/internal/PluginsMetadata.java

```diff
@@ -27,15 +27,32 @@
 
 import org.apache.maven.artifact.repository.metadata.Metadata;
 import org.apache.maven.artifact.repository.metadata.Plugin;
-import org.apache.maven.repository.internal.PluginsMetadataInfoProvider.PluginInfo;
-import org.eclipse.aether.artifact.Artifact;
 
 /**
- * Plugin G level metadata.
+ * Maven G level metadata.
  */
 final class PluginsMetadata
     extends MavenMetadata
 {
+    static final class PluginInfo
+    {
+        final String groupId;
```

Comment:
Why is this field not private? Maybe leave a comment for this?

Reply:
the whole class is pkg private, is shared by two classes from this package